app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,28 @@
+package com.globant.calculator.android;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
```

Comment:
Pls, remove this unnecessary spaces, thanks.

Reply:
Done, thanks